test/data/game_teams_test.csv

```diff
@@ -0,0 +1,3 @@
+game_id,team_id,HoA,result,settled_in,head_coach,goals,shots,tackles,pim,powerPlayOpportunities,powerPlayGoals,faceOffWinPercentage,giveaways,takeaways
+2012030221,1,away,LOSS,REG,John1 Doe1,2,8,44,8,3,0,44.8,17,7
+2012030221,2,home,WIN,RED,Claude Julien,3,12,51,6,4,1,55.2,4,5
```

Comment:
I'm assuming these would be the files we use for our tests since they're in the `./data` folder, it probably wouldn't hurt to have more of a sample size and also calling them `_test` files might get confusing as well

Reply:
Totally agree! what should we call the files instead of _test?

Reply:
*shrug* I used `_short` in my brief writing of code.. `_sample`?

Reply:
I like _sample. Push changes now